### ISSUE TITLE
Mouse does not commit changes

### ISSUE THREAD

**Issue by Kiedtl** [Kiedtl/mouse#7]
Mouse does not commit changes at some point, and as a result, Mouse crashes on `mouse backup when git-crypt refuses to unlock the repository and mouse tries to parse the encryptes json.

The output error  for mouse backup:

```
ERROR An unhandled exception was thrown in Mouse.
ERROR Please report the following error code:
INFO    Error string: exception@mouse(v1.1.4):[34~37~libexec/mouse-backup.ps1@entrypoint/cmd_exec]:[backup|]:[5.1.17134.407]:[Desktop::10.0.17134.0]::[NotSpecified: (:) [ConvertFrom-Json], ArgumentException]::[System.ArgumentException,Microsoft.PowerShell.Commands.ConvertFromJsonCommand]::[]::[C:\Users\*****]::[ConsoleHost]@incident[a6bd033d6ef75fca64e5bd587c272fe2202ff601]

```

I think this is caused when Mouse crashes and does not have time to commit changes elsewhere. I suggest adding a finally block to the main entrypoint so that is mouse crashes, it still commits any changes (if there are any) and locks the repository.

**Comment by Kiedtl** [comment]
Thid issue appears to be fixed by commit kiedtl@2d3a5844b8816cdb27c9dcaccdfb6f2b0ae7c71a. This issue will be reopened if similar behavior is observed elsewhere.

**Comment by Kiedtl** [comment]
Also, the finally block of code was implemented the commit kiedtl@5c925c427af179b4aba731a5db9fc4e89068453f.
